README.md

```diff
@@ -173,6 +174,8 @@
 
 - [lsp.v](https://github.com/nedpals/lsp.v) - Implementation of the Language Server Protocol spec on V.
 - [setup-v-action](https://github.com/marketplace/actions/setup-v-environment) - GitHub action automation to use V in your workflow.
+- [range](https://github.com/Delta456/range) - Functionality of Python's range() in V.
```

Comment:
I think this should be added in a separate section `Utility` .

Reply:
Agree. The `Other` section is for other stuff, not libraries.

Adding the new `Utility` subsection under the `Libraries` section makes more sense.

Reply:
Plus, `range()` could be surrounded by grave accents.

Reply:
Yes.

Reply:
I was thinking of the same things but didn't know what to name it
Thanks for the input

Reply:
Done! Created new category Utility and moved range to it ^^